examples/hypervisor_managers.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright [2019] Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+from config_loader import try_load_from_file
+from hpOneView.oneview_client import OneViewClient
+
+config = {
+    "ip": "",
+    "credentials": {
+        "userName": "",
+        "password": ""
+    }
+}
+
+hypervisor_manager_information = {
+    "type": "HypervisorManagerV2",
+    "name": "172.18.13.11",
+    "displayName": "vcenter",
+    "hypervisorType": "Vmware",
+    "username": "dcs",
+    "password": "dcs",
+    "initialScopeUris": []
+}
+
+# Try load config from a file (if there is a config file)
+config = try_load_from_file(config)
+oneview_client = OneViewClient.config()
```

Comment:
Please check the latest code changes: https://github.com/HewlettPackard/oneview-python/blob/master/CHANGELOG.md#breaking